src/cljs/witan/ui/strings.cljs

```diff
@@ -325,6 +327,8 @@
    :string.activity.add-file-to-datapack/completed      "You successfully added file '%s' to datapack '%s'"
    :string.activity.send-collect-request/failed         "You failed to send a collection request for datapack '%s'"
    :string.activity.send-collect-request/completed      "You successfully sent a collection request to %s recipient(s), for datapack '%s'"
+   :string.activity.add-files-to-datapack/failed "You failed to add files to the datapack."
+   :string.activity.add-files-to-datapack/completed "You successfully added files to the datapack."
```

Comment:
There are already failed and completed strings that look good (`string.activity.add-file-to-datapack/completed`) - can we use these?

Reply:
I don't think we can use those as they are formatted strings, C+S bundle add has no detail on the datapack name at this point as it's not loaded in for the view.

Reply:
You're correct - my bad